scripts/redcap/update_summary_scores

```diff
@@ -39,9 +42,35 @@ parser.add_argument("-n", "--no-upload",
 args = parser.parse_args()
 
 # First REDCap connection for the Summary project (this is where we put data)
-summary_key_file = open(os.path.join( os.path.expanduser("~"), '.server_config/redcap-dataentry-token'), 'r')
-summary_api_key = summary_key_file.read().strip()
-rc_summary = redcap.Project('https://ncanda.sri.com/redcap/api/', summary_api_key, verify_ssl=False)
+def connect_to_redcap():
+    summary_key_file = open(os.path.join( os.path.expanduser("~"), '.server_config/redcap-dataentry-token'), 'r')
+    summary_api_key = summary_key_file.read().strip()
+    rc = redcap.Project(redcap_server, summary_api_key, verify_ssl=False)
+    return rc
+
+# If connection to redcap server fail, try multiple times
```

Comment:
same thing here. I vote that we remove this 'retry' code and just catch the exception